README.md

```diff
@@ -28,16 +28,14 @@ A PC with *Windows 7 or above*, A smartphone with *Android 5.0(Lollipop) or abov
 ## Installation
 
 * Make sure you have Steam and SteamVR installed (To find SteamVR on steam, `Library -> Tools -> SteamVR`).
-* Download latest [`PhoneVR.zip`](https://github.com/ShootingKing-AM/PhoneVR/releases) release of this repository.
+* Download latest release build [`PhoneVR-0.1-beta_41-master.zip`](https://github.com/ShootingKing-AM/PhoneVR/releases/latest) release of this repository.
```

Comment:
Dont hardcode .zip/PhoneVR version in Readme, since this would be changing for every build.

Reply:
i know, its just for example.

Reply:
You can commit the changes, and mark this request as resolved.